src/GetStartDate.js

```diff
@@ -0,0 +1,12 @@
+import moment from 'moment';
+
+const END_DATE = moment().subtract(1, 'days').toDate();
+
+export default function GetStartDate() {
```

Comment:
This fixed it! No changes needed in the code for this section. I'd suggest adding short comments here in case the client needs to understand what is going